src/applications/vaos/appointment-list/components/cards/confirmed/ConfirmedAppointmentListItem.jsx

```diff
@@ -107,12 +115,25 @@ export default function ConfirmedAppointmentListItem({
         {header}
       </div>
       <h3 className="vaos-appts__date-time vads-u-font-size--h3 vads-u-margin-x--0">
+        {isAtlasLocation(appointment) && (
```

Comment:
This is somewhat of a nitpick but since you're using it multiple times, you may want to declare 
```js
const isAtlas = isAtlasLocation(appointment);
```
up above and reuse it below.

Reply:
Done.